cli/ops.rs

```diff
@@ -33,10 +33,7 @@ use deno::Op;
 use deno::PinnedBuf;
 use flatbuffers::FlatBufferBuilder;
 use futures;
-use futures::Async;
-use futures::Poll;
-use futures::Sink;
-use futures::Stream;
+use futures::{future, Async, Poll, Sink, Stream};
```

Comment:
Again, against our coding style, please don't do this.

Reply:
I'll fix this